packages/components/file-size/src/file-size.js

```diff
@@ -0,0 +1,58 @@
+import { TSElement, unsafeCSS, html } from '@tradeshift/elements';
+import '@tradeshift/elements.typography';
+
+import css from './file-size.css';
+
+customElements.define(
+	'ts-file-size',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				size: { type: Number, reflect: true },
+				decimalPoint: {
+					type: Number,
+					reflect: true,
+					attribute: 'decimal-point'
+				},
+				variant: { type: String, reflect: true },
+				color: { type: String, reflect: true }
+			};
+		}
+
+		constructor() {
+			super();
+			this.size = 0;
+			this.decimalPoint = 2;
+			this.variant = 'subtitle';
+			this.color = 'default';
+		}
+
+		get formatFileSize() {
+			const bytes = this.size;
+			const decimalPoint = this.decimalPoint;
+			if (bytes === 0) return '0 Bytes';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s) in this "if" block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACurlyBraces)